cumulus_etl/upload_notes/cli.py

```diff
@@ -240,11 +247,9 @@ def group_notes_by_encounter(notes: Collection[LabelStudioNote]) -> list[LabelSt
                 match.end += offset
                 grouped_ctakes_matches.append(match)
 
-            for label, spans in note.highlights.items():
-                grouped_highlights[label] = []
```

Comment:
This line here was a nasty bug I missed - it would remove highlighted terms from previous docs in the group for this label.